media-libs/libvmaf/libvmaf-2.3.0.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MULTILIB_COMPAT=( abi_x86_{32,64} )
```

Comment:
... is this actually needed?

Reply:
Okay, so I removed that and simply inherited `meson` eclass, basically [this version](https://github.com/gentoo/gentoo/blob/85931e308c9fa4ce7aaca166cf41c29863fda040/media-libs/libvmaf/libvmaf-2.3.0.ebuild). And my `ffmpeg` that is built with multilib support fails.

Or are you saying I can still inherit `meson-multilib` and not set `MULTILIB_COMPAT=( abi_x86_{32,64} )` ?

Reply:
Yeah now that you mention it, it makes sense so other arches aren't just blocked because of this arbitrary restriction. done!